model/SolutionManager.py

```diff
@@ -1,19 +1,39 @@
-import model.SolutionInstance
-from model.SolutionInstance import *
-from tools.JSonExportUtility import *
+# Copyright (c) 2020 Aalto University. All rights reserved.
 
-hashToSolution = dict()
+from model.SolutionInstance import SolutionInstance
+from tools.JSONExportUtility import save_to_json
+from model.DataInstance import DataInstance
+from tools.PlotUtils import draw_solution
 
-def buildNewSolution(objValue, Lval,Tval, Wval, Hval):
-    solution = SolutionInstance(objValue, Lval,Tval, Wval, Hval)
-    hash = str(Lval)+str(Tval)+ str(Wval)+str(Hval)
-    if hash in hashToSolution:
-        print("** Neglecting a repeat solution **")
-        return
-    else:
-        hashToSolution[hash] = solution
-        useSolution(objValue, Lval,Tval, Wval, Hval)
 
+class SolutionManager:
```

Comment:
Now a class as well so that code can be kept running and no global state is used to determine repeated solutions